core/include/mm/pgt_cache.h

```diff
@@ -66,28 +67,23 @@ bool pgt_check_avail(struct pgt_cache *pgt_cache, struct vm_info *vm_info);
  * needed translation tables.
  */
 #if defined(CFG_CORE_PREALLOC_EL0_TBLS)
-static inline void pgt_get_all(struct pgt_cache *pgt_cache __unused,
-			       struct ts_ctx *owning_ctx __unused,
-			       struct vm_info *vm_info __unused) { }
+static inline void pgt_get_all(struct user_mode_ctx *uctx __unused) { }
 #else
-void pgt_get_all(struct pgt_cache *pgt_cache, struct ts_ctx *owning_ctx,
-		 struct vm_info *vm_info);
+void pgt_get_all(struct user_mode_ctx *uctx);
 #endif
 /*
  * pgt_put_all() - informs the translation table manager that these tables
  *		   will not be needed for while
  * @pgt_cache:	list of translation table to put to rest
  */
 #if defined(CFG_CORE_PREALLOC_EL0_TBLS)
-static inline void pgt_put_all(struct pgt_cache *pgt_cache __unused) { }
+static inline void pgt_put_all(struct user_mode_ctx *uctx __unused) { }
 #else
-void pgt_put_all(struct pgt_cache *pgt_cache);
+void pgt_put_all(struct user_mode_ctx *uctx);
```

Comment:
[commit "core: simplify pgt interface provided by pgt_cache.h"]

The comment needs updating to reflect the new signature.